budgetportal/summaries.py

```diff
@@ -474,54 +485,67 @@ def get_dataset(self):
         return self.department.get_estimates_of_subprogramme_expenditure_dataset()
 
     def get_aggregate_cuts(self):
-        openspending_api = self.get_openspending_api()
-        financial_year_start = self.department.get_financial_year().get_starting_year()
-        cuts = [
-            openspending_api.get_financial_year_ref() + ":" + financial_year_start,
-            openspending_api.get_department_name_ref() + ":" + self.department.name,
-            openspending_api.get_phase_ref() + ":" + "Main appropriation",
-        ]
-        if self.department.government.sphere.slug == "provincial":
-            cuts.append(
-                openspending_api.get_geo_ref()
-                + ':"%s"' % self.department.government.name
-            )
-        return cuts
+        if self.get_openspending_api is not None:
+            openspending_api = self.get_openspending_api()
+            financial_year_start = self.department.get_financial_year().get_starting_year()
+            cuts = [
+                openspending_api.get_financial_year_ref() + ":" + financial_year_start,
+                openspending_api.get_department_name_ref() + ":" + self.department.name,
+                openspending_api.get_phase_ref() + ":" + "Main appropriation",
+            ]
+            if self.department.government.sphere.slug == "provincial":
+                cuts.append(
+                    openspending_api.get_geo_ref()
+                    + ':"%s"' % self.department.government.name
+                )
+            return cuts
+        else:
+            return None
 
     def get_aggregate_drilldowns(self):
-        openspending_api = self.get_openspending_api()
-        return [
-            openspending_api.get_programme_name_ref(),
-            openspending_api.get_subprogramme_name_ref(),
-        ]
+        if self.get_openspending_api is not None:
+            openspending_api = self.get_openspending_api()
+            return [
+                openspending_api.get_programme_name_ref(),
+                openspending_api.get_subprogramme_name_ref(),
+            ]
+        else:
+            return None
```

Comment:
another, perhaps easier to read, way to write this

```suggestion
        if self.get_openspending_api is None:
            return None
        else:
            openspending_api = self.get_openspending_api()
            return [
                openspending_api.get_programme_name_ref(),
                openspending_api.get_subprogramme_name_ref(),
            ]
```